services/core-web/src/components/noticeOfWork/applications/PermitPackage.js

```diff
@@ -0,0 +1,287 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Button, notification } from "antd";
+import { DownloadOutlined } from "@ant-design/icons";
+
+import { openModal, closeModal } from "@common/actions/modalActions";
+import { getDocumentDownloadToken } from "@common/utils/actionlessNetworkCalls";
+import { modalConfig } from "@/components/modalContent/config";
+import CustomPropTypes from "@/customPropTypes";
+import {
+  createNoticeOfWorkApplicationReview,
+  fetchNoticeOfWorkApplicationReviews,
+  deleteNoticeOfWorkApplicationReview,
+  updateNoticeOfWorkApplicationReview,
+  deleteNoticeOfWorkApplicationDocument,
+  setNoticeOfWorkApplicationDocumentDownloadState,
+  updateNoticeOfWorkApplication,
+  fetchImportedNoticeOfWorkApplication,
+} from "@common/actionCreators/noticeOfWorkActionCreator";
```

Comment:
has this file been linted correctly? These functions should be listed in PropTypes.

Reply:
I did lint this and got no errors but I will take another look at it

Reply:
I removed the unused ones but they were not throwing a linting error